python/ql/src/Security/CWE-327/InsecureDefaultProtocol.ql

```diff
@@ -10,21 +10,20 @@
  *       external/cwe/cwe-327
  */
 
+// Connections are generally created based on a context which controls the range of acceptable
+// protocols. This query reports the deprecated way of creating connections without referring
+// to a context (via `ssl.wrap_socket`). Doing this and not specifying which protocols are
+// acceptable means that connections will be created with the insecure default settings.
+//
+// Detecting that a connection is created with a context that has not been suitably configured
+// is handled by the data-flow query py/insecure-protocol.
 import python
-
-FunctionValue ssl_wrap_socket() { result = Value::named("ssl.wrap_socket") }
-
-ClassValue ssl_Context_class() { result = Value::named("ssl.SSLContext") }
+import semmle.python.ApiGraphs
 
 CallNode unsafe_call(string method_name) {
```

Comment:
With all these rewrites, there's really no reason to put this in a separate utility predicate anymore. I would suggest simply inlining the body of this predicate into the `where` clause, and to also use `CallCfgNode` instead of `CallNode`.
That's assuming we're not planning on adding other insecure defaults, but judging by the discussion, this seems to not be the case.

Reply:
Agreed, that would also eliminate the variable `method_name`.

Reply:
In fact, that became a fair bit nicer :-)